src/query/storages/parquet/src/parquet_rs/parquet_reader/row_group.rs

```diff
@@ -96,7 +96,7 @@ pub struct InMemoryRowGroup<'a> {
     metadata: &'a RowGroupMetaData,
     page_locations: Option<&'a [Vec<PageLocation>]>,
     column_chunks: Vec<Option<Arc<ColumnChunkData>>>,
-    row_count: usize,
+    pub row_count: usize,
```

Comment:
I think it's better to expose a public method instead of exposing a public member.